src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.cs

```diff
@@ -0,0 +1,210 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+namespace System.Threading
+{
+    public static partial class ThreadPool
+    {
+        internal static bool UseWindowsThreadPool
+        {
+            get => AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.UseWindowsThreadPool", "DOTNET_ThreadPool_UseWindowsThreadPool");
```

Comment:
This is going to fetch the config string by name every time this property is called. You do not want that - it would be very slow. 

You want to use auto-initialized property backed by a field:
`.. { get; } = AppContextConfigHelper...` . It is going to fetch the value once, and then keeps it cached.